src/app/shared/services/app.service.ts

```diff
@@ -177,6 +177,17 @@ export class AppService implements OnDestroy {
           this._initAppError(error.message);
         }
       );
+
+    // Load jQuery to support player loading long configuration string (more than 2083 characters)
+    if (analyticsConfig.previewPlayer.loadJquery && document.getElementById("jquery") === null) {
+      let jq = document.createElement('script');
+      jq.src = 'https://code.jquery.com/jquery-3.5.1.min.js';
```

Comment:
Please make sure to load jQuery from a kaltura location, as in Air Gapped On Prems, we cannot load it from the internet